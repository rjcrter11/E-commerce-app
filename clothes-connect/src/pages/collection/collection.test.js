import React from 'react';

import { shallow } from 'enzyme';

import { CollectionPage } from './collection.component'
//import * as collectionItem from '../../components/collection-item'
describe('CollectionPage component', () => {
    let enzymeWrapper;
    let mockItems = [{ id: 1 }, { id: 2 }, { id: 3 }];

    beforeEach(() => {
        const mockCollection = {
            items: mockItems,
            title: 'test'
        };


        enzymeWrapper = shallow(<CollectionPage collection={mockCollection} />)

    })

    it('should render CollectionPage component', () => {
        expect(enzymeWrapper).toMatchSnapshot()
    })
})