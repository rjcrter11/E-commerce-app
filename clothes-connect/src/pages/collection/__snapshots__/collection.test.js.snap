// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionPage component should render CollectionPage component 1`] = `ShallowWrapper {}`;
